pkg/core/alias.go

```diff
@@ -35,8 +35,8 @@ var (
 		}()
 		return ctx
 	}
-
-	NewUUID = func() string {
-		return uuid.New().String()
-	}
 )
+
+func NewUUID() string {
+	return uuid.New().String()
```

Comment:
With github.com/google/uuid 1.3.0, you can replace every `uuid.New().String()`with `uuid.NewString()`

See : https://pkg.go.dev/github.com/google/uuid#NewString